docker/nginx-default.conf

```diff
@@ -0,0 +1,41 @@
+server {
+    listen 3001;
+
+    location / {
+        include cors.conf;
+        proxy_pass http://node1:3013/;
```

Comment:
I noticed that, after having started the localnet (`docker-compose up -d`) and checked they were all reachable via curl via proxy, when I stopped all containers one by one (`docker stop epoch_node1_1`, `docker stop epoch_node2_1`, `docker stop epoch_node3_1`) and then I started the first one (`docker start epoch_node3_1`), this started container was not reachable via curl via proxy ("No route to host"). Then I started the other two nodes too and all 3 were reachable via curl.

I expect this glitch is related to Docker (Docker for Mac "Docker version 17.12.0-ce, build c97c6d6") and not actionable on our side. Something to keep in mind if users report this kind of behaviour.

Reply:
Well, this setup intended usage is via `docker-compose` only, if you hack around with the containers itself with `docker` command only you're responsible yourself to setup the networking and port mapping.